test/integration/test/gangway_test.go

```diff
@@ -515,24 +516,22 @@ func TestGangwayBulkJobStatusChange(t *testing.T) {
 			}
 			t.Logf("Created %d jobs", len(pjs.Items))
 
-			jobsAffected, err := c.GRPC.BulkJobStatusChange(ctx, tt.bulkMsg)
+			_, err = c.GRPC.BulkJobStatusChange(ctx, tt.bulkMsg)
 			if tt.expectedErr != nil {
 				if !errors.Is(err, tt.expectedErr) {
 					t.Fatalf("Expected error %v, got %v", tt.expectedErr, err)
 				}
 			} else {
-				if jobsAffected.Count != int32(tt.expectedCount) {
-					t.Fatalf("Expected %d jobs to be affected, got %d", tt.expectedCount, jobsAffected.Count)
-				}
-				for _, job := range jobsAffected.JobExecutions {
-					if job.JobStatus != gangway.JobExecutionStatus_ABORTED {
-						t.Fatalf("Expected job status to be %q, got %q", gangway.JobExecutionStatus_ABORTED, job.JobStatus)
+				for _, jobExecution := range jobExecutions {
+					timeout := 120 * time.Second
```

Comment:
how did you decide on the timeout? that seems like a long time to wait during an integration test, does it actually take that long?

Reply:
No, it doesn't take long, I copied it from the test above